.env.example

```diff
@@ -35,6 +35,25 @@ BH_POSTGRES_DB=bloodhound
 BH_POSTGRES_VOLUME=bh-postgres-data
 BH_POSTGRES_PORT=127.0.0.1:6543
 
+# BloodHound Authentik SSO IdP
+BH_PG_USER=authentik-bloodhound
+BH_PG_DB=authentik-bloodhound
+BH_PG_PASS=bloodhoundcommunityedition
+BH_ATK_SECRET=bloodhoundcommunityedition
+COMPOSE_PORT_HTTP=127.0.0.1:9000
+COMPOSE_PORT_HTTPS=127.0.0.1:9443
+
+# BloodHound Authentik Worker
+BH_PG_USER=authentik-bloodhound
+BH_PG_DB=authentik-bloodhound
+BH_PG_PASS=bloodhoundcommunityedition
+BH_ATK_SECRET=bloodhoundcommunityedition
+
+# BloodHound Authentik Postgres
+BH_PG_PASS=bloodhoundcommunityedition
+BH_PG_USER=authentik-bloodhound
+BH_PG_DB=authentik-bloodhound
+
```

Comment:
Why are all of these env vars being duplicated? Also, I'd probably recommend using the ATK prefix for all related vars. Putting them as PG makes them easier to confuse with BH_POSTGRES vars

Reply:
They're duplicated since a lot of those same env variables are used in different authentik container services. Yeah I'll def change them to ATK to not get confused. I can see how things would get mixed up